iamidentityv1/iam_identity_v1_integration_test.go

```diff
@@ -604,6 +604,7 @@ var _ = Describe(`IamIdentityV1 Integration Tests`, func() {
 				Mfa:                          core.StringPtr("NONE"),
 				SessionExpirationInSeconds:   core.StringPtr("86400"),
 				SessionInvalidationInSeconds: core.StringPtr("7200"),
+				MaxSessionsPerIdentity:       core.StringPtr("10"),
```

Comment:
@padamstx Can we skip adding this parameter to the examples or we should do it?

Reply:
let's add it to examples